src/lib/sidenav/sidenav.md

```diff
@@ -70,3 +70,19 @@ html, body, material-app, md-sidenav-container, .my-content {
 ### FABs inside sidenav
 For a sidenav with a FAB (Floating Action Button) or other floating element, the recommended approach is to place the FAB
 outside of the scrollable region and absolutely position it.
+
+### Sidenav vs. Drawer
```

Comment:
I would put this at the top since its important context that users need to understand the rest of the docs (that all the stuff we're saying about sidenavs also applies to drawer